mockturtle/README.md

```diff
@@ -0,0 +1,28 @@
+# mockturtle.rb
+
+The Ruby based IRC bot which runs in [`#lobsters`](https://lobste.rs/chat). 
+
+## Running
+
+```sh
+$ ruby app/mockturtle.rb 'irc://username:password@example.com:6697/#channel1,##channel2`
+```
+
+## License
+
+```
+Copyright 2025 Noah Hendrickson
```

Comment:
I'd prefer to use the license that's already used for the site (3 clause BSD, if my quick glance is correct) because code will be shared between the bot and site. Do you have a strong preference that we do it differently?

Reply:
No strong preference, will copy.